maps-compose-utils/src/main/java/com/google/maps/android/compose/kml/manager/ContainerManager.kt

```diff
@@ -0,0 +1,93 @@
+package com.google.maps.android.compose.kml.manager
+
+import android.content.Context
+import android.graphics.Bitmap
+import androidx.compose.runtime.Composable
+import com.google.maps.android.compose.kml.data.KmlStyle
+import com.google.maps.android.compose.kml.data.KmlStyleMap
+
+public class ContainerManager() : KmlComposableManager {
+    private var containerName: String = ""
+    override var style: KmlStyle = KmlStyle()
+    private val containers: MutableList<ContainerManager> = mutableListOf()
+    private val markers: MutableList<MarkerManager> = mutableListOf()
+
+    public fun getName(): String = containerName
+
+    /**
+     * Gets a list of containers that are direct children of this ContainerManager
+     *
+     * @return List of [ContainerManager]s
+     */
+    public fun getContainers(): List<ContainerManager> = containers
+
+    /**
+     * Gets a list of markers that are direct children of this ContainerManager
+     * @return List of [MarkerManager]s
+     */
+    public fun getMarkers(): List<MarkerManager> = markers
+
+    /**
+     *  Sets properties from KML relevant to the container
+     *
+     *  @param data HashMap containing related properties of the container
+     */
+    override fun setProperties(data: HashMap<String, Any>) {
+        //TODO()
```

Comment:
todo, is dit iets wat nu nog moet gebeuren?

Reply:
Nee de todos zijn voor volgende taken als geheugensteun